### PR TITLE
pico_ultrasonic_ranger 0.1.0

### PR DESCRIPTION
**Pull request by KLOC-Karsten**
Created via `alr publish` with `alr 2.0.2+9b80158`

### CODE REVIEW

File: index/pi/pico_ultrasonic_ranger/pico_ultrasonic_ranger-0.1.0.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name = "pico_ultrasonic_ranger"
+description = "Driver for the Grove ultrasonic ranger. Depends on the RP Pico HAL."
```
Comment (mosteo) [inline index/pi/pico_ultrasonic_ranger/pico_ultrasonic_ranger-0.1.0.toml:2]: Would it be better that `grove` appears in the crate name? E.g. `grove_pico_ultrasonic_ranger`

Comment (KLOC-Karsten) [reply index/pi/pico_ultrasonic_ranger/pico_ultrasonic_ranger-0.1.0.toml:2]: I thought of that, but the driver may work also with the Adafruit or Sparkfun rangers, therefore I decided not to add Grove in the crate name.

Comment (mosteo) [reply index/pi/pico_ultrasonic_ranger/pico_ultrasonic_ranger-0.1.0.toml:2]: OK!

**Review by mosteo — APPROVED** [review]

**Comment by mosteo** [comment]
This is still in draft mode, OK to merge?

**Pull request merged**